andela/ET-cfh
# #158457041 Users should be quickly and effectively onboarded onto the app

#### What does this PR do?
It creates a tour to onboard new users to the gaming screen when they want to play a game

#### Description of Task to be completed?
Create a tour for the gaming screen and make the tour available to only new users when they first go to the gaming screen

#### How should this be manually tested?
https://et-cfh-staging-pr-20.herokuapp.com/#!/app
Login to your account and navigate to the gaming screen. It should give you a tour the first time

#### What are the relevant pivotal tracker stories?
#158457041

#### Screenshots
<img width="1440" alt="screen shot 2018-07-17 at 10 28 06 pm" src="https://user-images.githubusercontent.com/32465536/42846593-4bb5a2f4-8a11-11e8-9b8c-925a83c1db86.png">
<img width="1439" alt="screen shot 2018-07-17 at 10 28 28 pm" src="https://user-images.githubusercontent.com/32465536/42846601-4e119792-8a11-11e8-886a-b2c1748a9cc5.png">
